src/client/Microsoft.Identity.Client/AppConfig/ConfidentialClientApplicationBuilder.cs

```diff
@@ -174,6 +180,38 @@ public ConfidentialClientApplicationBuilder WithClientAssertion(Func<string> cli
             return this;
         }
 
+
+        /// <summary>
+        /// Instructs MSAL to use an Azure regional token service using the region given.
+        /// 
+        /// If the calling app knows the region it is deployed to, it should use this information. 
+        /// 
+        /// Otherwise, set the variable to <see cref="ConfidentialClientApplication.AttemptRegionAutoDiscovery"/>, and MSAL will attempt to auto-detect the region. This process
+        /// works on a limited number of Azure artifacts (TBD - which ones!?). If auto-discovery fails, MSAL will use the non-regional service.
+        ///         
+        /// See https://aka.ms/msal-net-region-discovery for more details.
+        /// </summary>
+        /// <param name="azureRegion">A string indicating the azure region, as per https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.management.resourcemanager.fluent.core.region?view=azure-dotnet
+        /// Or <see cref="ConfidentialClientApplication.AttemptRegionAutoDiscovery"/> to have MSAL auto-detect the region.
```

Comment:
```suggestion
        /// Or <see cref="ConfidentialClientApplication.AttemptRegionAutoDiscovery"/> to have MSAL auto-detect the region (which is the default value).
```

Reply:
It's clear from the IDE that it's the default value